src/TimeTracker.Library/Services/Interpretation/ReportInterpreter.cs

```diff
@@ -7,12 +7,22 @@
 
 namespace TimeTracker.Library.Services.Interpretation
 {
+    public enum ReportType
```

Comment:
thanks for adding this enum to make parsing clearer [JR]

Reply:
Nate's great idea!